<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="UDIND-LP041" failures="1" tests="2" name="Default test(failed)" time="0.066" errors="0" timestamp="2022-06-14T18:22:19 IST">
  <testcase classname="Day11.TestNGRetryFailedDemo" name="test3" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Day11.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="Day11.TestNGRetryFailedDemo" name="test2" time="0.002"/>
  <testcase classname="Day11.TestNGRetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
